src/packages/cli/src/cli.ts

```diff
@@ -141,6 +149,8 @@ async function startGanache(err: Error) {
       break;
     }
   }
+  versionCheck.init().log();
+  versionCheck = null;
```

Comment:
Setting this to `null` here prevents the `closeHandler` from calling `versionCheck.destroy()` even if `versionCheck` is still doing work in the background.

Reply:
:+1: https://github.com/trufflesuite/ganache/pull/3285/commits/0644054b8458eafdb52f2a9699842ed0c91f6a4e

Reply:
I don't know how or why, but this `versionCheck = null;` came back after you removed it